### PR TITLE
Mtk 2 0 0 release

### PR DESCRIPTION
**Pull request by jbech-linaro**
This PR contains the two commits in #54 and one additional patch on top of that. Everything has been rebased to latest. If/when this gets merged, then we can just close #54 .

### CODE REVIEW

File: kconfigs/mediatek.conf [hunk 1]
```diff
@@ -1 +1,4 @@
-CONFIG_DRM=y
+CONFIG_TEE=y
+CONFIG_OPTEE=y
+### This prevents random failures of "make check" in Travis CI
+CONFIG_PREEMPT=y
```
Comment (jenswi-linaro) [inline kconfigs/mediatek.conf]: I think this is the default for arm64 already.

Comment (jbech-linaro) [inline kconfigs/mediatek.conf]: Let me double check and if yes, then I'll remove it.

**Comment by jenswi-linaro** [comment]
`Reviewed-by: Jens Wiklander <jens.wiklander@linaro.org>`

**Comment by pascal-brand-st-dev** [comment]
`Reviewed-by: Pascal Brand <pascal.brand@linaro.org>`

File: kconfigs/mediatek.conf (hunk 1 above)
Comment (jbech-linaro) [inline kconfigs/mediatek.conf]: You're right, it's there, I'll remove it, add the tags, and force push the update.

**Comment by jbech-linaro** [comment]
Patch rebased, tags applied, ready to be merged.

**Pull request merged**